src/main/scala/net/jcazevedo/moultingyaml/YamlValue.scala

```diff
@@ -16,11 +15,9 @@ sealed abstract class YamlValue {
 
   private[moultingyaml] def snakeYamlObject: Object
 
-  def prettyPrint: String = {
-    val options = new DumperOptions()
-    options.setDefaultFlowStyle(DumperOptions.FlowStyle.BLOCK)
-    val yaml = new Yaml(options)
-    yaml.dump(snakeYamlObject)
+  def prettyPrint(flowStyle: FlowStyle = Block(), scalarStyle: ScalarStyle = Plain()): String = {
```

Comment:
I don't like to have a configurable `prettyPrint` method. For me, this method would create a `SnakeYAMLPrinter` with `Block` flow style and plain scalar style and use it to print the document.

We can add a `print` method to `YamlValue` that allows customization, though.